torch/quantization/fake_quantize.py

```diff
@@ -144,4 +151,4 @@ def disable_observer(mod):
 
 def enable_observer(mod):
     if type(mod) == FakeQuantize:
-        mod.disable_observer()
+        mod.enable_observer()
```

Comment:
lol :)

Reply:
This is why we need tests for every single line of code change :-)